### ISSUE TITLE
reboot server, NC is fine for like 20 min then internal server error

### ISSUE THREAD

**Issue by ihaveimpulses** [YunoHost-Apps/nextcloud_ynh#591]
### Describe the bug

I want to be as helpful as possible here since obviously I need help, but I might have some issues with knowledge, like I can't find a log for nextcloud in YNH even though the error asks for it. 

Long story short yesterday and in the past all day everyday nextcloud was/is fine. 

I wake up to errors on my phone with syncing calendar so I try to foto the site and get
Internal Server Error

The server encountered an internal error and was unable to complete your request.
Please contact the server administrator if this error reappears multiple times, please include the technical details below in your report.
More details can be found in the server log.

So I go down to my server and reboot it. Nextcloud works fine, I go up and shower and come down and check it:
Internal Server Error

The server encountered an internal error and was unable to complete your request.
Please contact the server administrator if this error reappears multiple times, please include the technical details below in your report.
More details can be found in the server log.

I reboot again and it's fine. I go make coffee and site down and check it:
Internal Server Error

The server encountered an internal error and was unable to complete your request.
Please contact the server administrator if this error reappears multiple times, please include the technical details below in your report.
More details can be found in the server log.

seems every 20 or so min it dies. Everything else YNH is working fine, mattermost, wordpress, vaultwarden, etc. 

### Context

- Hardware: bare metal home pc. 8TB SSD, AMD 5700G, 128GB of RAM
- YunoHost version:11.2.3 (stable)

**Comment by Sonnenblumenkind** [comment]
Have you installed the "Recognize" App? If so, remove it. It can completely saturate your database and then Nextcloud is gone.

**Comment by aquaspy** [comment]
The fix for this is: https://github.com/linuxserver/docker-nextcloud/issues/211

TL;DR:
Modify the config/nginx/site-confs/default file as follows. The new lines should be added in the location ~ \.php(?:$|/) section:

Add
```
fastcgi_read_timeout 600;
fastcgi_send_timeout 600;
fastcgi_connect_timeout 600;
proxy_connect_timeout 600;
proxy_send_timeout 600;
proxy_read_timeout 600;
send_timeout 600;```

Save

**Comment by aquaspy** [comment]
@ericgaspar thank you so much for the pull request!

**Comment by alexAubin** [comment]
Assuming it was fixed in https://github.com/YunoHost-Apps/nextcloud_ynh/pull/596
